src/Validator/ErrorElement.php

```diff
@@ -256,6 +256,17 @@ private function newConstraint(string $name, array $options = []): Constraint
             ));
         }
 
+        $reflectionClass = new \ReflectionClass($className);
```

Comment:
@VincentLanglet are you familiar with this API? Like calling those magic `assertXYZ` methods on `ErrorElement`. Where is this done or whats the purpose?

Wondering about performance implications for using reflection here :thinking: 

Checked and in my app testsuite this is never called. So not sure about the use-case for this. Supposed to be called only in user-land code?

Reply:
No idea about the use case.

This is something which was considered to be deprecated https://github.com/sonata-project/form-extensions/issues/333

But it's used in some other bundle like https://github.com/sonata-project/SonataMediaBundle/blob/426f0689be5e512e5fb9e9b733cd4db997af1a1b/src/Provider/MediaProviderInterface.php#L186

Reply:
Thanks! I will see if I can take a look at it some time soon. Would be good to get rid of it